initramfs/bootscript

```diff
@@ -3,16 +3,23 @@
 # and the state attested.  The secret file from the remote attestation server
 # will be passed on the command line
 
-#set -x
+set -euo pipefail
+
+echo "TPM2TOOLS_TCTI=$TPM2TOOLS_TCTI"
+
+[[ -n ${BOOTSCRIPT_DEBUG:-} ]] && set -vx
+
 POLICY_PCR=11
 die() {
+	set +euo pipefail
```

Comment:
why are the shell flags adjusted here?

Reply:
Habit.  I tend to do this in EXIT trap handlers and such to avoid bugs in them causing partial cleanup.  I can remove this.

Reply:
Fixed.